src/ribbon/gallery.cpp

```diff
@@ -543,7 +544,7 @@ wxRibbonGalleryItem* wxRibbonGallery::Append(const wxBitmap& bitmap, int id)
 
     wxRibbonGalleryItem *item = new wxRibbonGalleryItem;
     item->SetId(id);
-    item->SetBitmap(bitmap);
+    item->SetBitmap(bitmap.IsOk() ? bitmap : wxBitmap{});
```

Comment:
This doesn't do anything. "Not ok" is the same as "empty".

Reply:
Rolled that back.